src/openrpc/media-info.json

```diff
@@ -275,6 +273,41 @@
                     "unknown",
                     "none"
                 ]
+            },
+            "MimeType": {
+                "type": "string",
+                "description": "Enumeration of possible Mimetypes.",
+                "enum": [
```

Comment:
Do methods returning MIME types need to respond with the internal name (AUDIO_AC3) or value (audio/ac3)?